src/replication/log_consumer.cpp

```diff
@@ -82,6 +82,9 @@ namespace cubreplication
 
       void execute (cubthread::entry &thread_ref) final
       {
+	assert (!m_repl_stream_entries.empty () && m_repl_stream_entries.back ()->is_group_commit ());
```

Comment:
Are you sure the last replication object must be a group commit?

Reply:
Fixed :)